pkg/metrics/metrics.jsx

```diff
@@ -1381,8 +1386,11 @@ const PCPConfigDialog = ({
     const handleInstall = async () => {
     // when enabling services, install missing packages on demand
         const missing = [];
-        if (dialogLoggerValue && !s_pmlogger.exists)
-            missing.push("cockpit-pcp");
+        let pcp_missing = false;
+        if (dialogLoggerValue && !s_pmlogger.exists) {
+            missing.push(await get_pcp_packagelist());
```

Comment:
So this assumes that pcp always depends on python3-pcp, right? i.e `!s_pmlogger.exists` is a sufficient condition.